Make event names in slack notifications consistent

arun1595 opened the pull request:
<!--
  Thank you for your pull request. Please provide a description above and review
  the requirements below.

  Contributors guide: https://github.com/gitcoinco/web/blob/contributing/CONTRIBUTING.md
-->

##### Description
<!-- A description on what this PR aims to solve -->

The event names **START WORK** and **STOP WORK** used in slack notifications don't describe the actual events which are work being started and work being stopped (See screenshot). This PR updates the event names to make it consistent with the rest of the events (WORKER APPROVED, WORKER REJECTED)

![image](https://user-images.githubusercontent.com/7039523/42261742-c43fcd78-7f2e-11e8-9862-50ff861c698f.png)

##### Checklist
<!-- Remove items that do not apply. For completed items, change [ ] to [x]. -->

- [x] linter status: 100% pass
- [x] changes don't break existing behavior
- [x] commit message follows [commit guidelines](https://github.com/gitcoinco/web/blob/master/docs/CONTRIBUTING.md#step-4-commit)

arun1595 commented on app/dashboard/notifications.py:
```diff
@@ -213,7 +213,7 @@ def build_message_for_integration(bounty, event_name):
 
     title = bounty.title if bounty.title else bounty.github_url
-    msg = f"*{humanize(event_name.replace('bounty', 'funded_issue'))}*" \
+    msg = f"*{humanize_event_name(event_name))}*" \
```
I hate this `event_name` duplication but I couldn't come up with a better name :disappointed:

mbeacom replied:
This fails travis because there is an extra `)` here.

PixelantDesign commented:
@mbeacom can you merge this?

mbeacom commented on app/dashboard/utils.py:
```diff
@@ -55,15 +55,29 @@ class NoBountiesException(Exception):
     """
-    return text.replace('_', ' ').upper()
+    humanized_event_names = {
+      'new_bounty': 'New funded issue',
```
This should be 4 space indentation.

mbeacom commented on app/dashboard/utils.py:
```diff
@@ -55,15 +55,29 @@ class NoBountiesException(Exception):
 
-def humanize(text):
-    """Clean a notification string of underscores and capitalize it.
+def humanize_event_name(name):
```
Can you write a test for this method in `dashboard/tests/test_dashboard_utils.py` ?

mbeacom commented:
@PixelantDesign Travis CI / pytest is failing. As soon as the requested changes are turned around, yes.

arun1595 commented:
@mbeacom Sorry about the syntax error, it was totally my bad. I've fixed the indentation issue and also added a test for `humanize_event_name`. Can you PTAL?

mbeacom commented:
Travis is still failing here - Probably has to do with a previous `master` that was failing... Updating from the latest master.  If builds succeed, will merge.

Thanks

arun1595 commented:
@mbeacom 
> if you could remove the [WOC] from the GH PR title after turning over the review comments, it'll greatly accelerate the followup review

Ah, will keep in mind. Thanks for getting this merged!
